src/NServiceBus.AcceptanceTests/DataBus/When_sending_databus_properties_with_unobtrusive.cs

```diff
@@ -0,0 +1,99 @@
+                {
+                    Payload = payloadToSend
+                })))
+                .WithEndpoint<Receiver>()
+                .Done(c => c.ReceivedPayload != null)
+                .Run();
+
+            Assert.AreEqual(payloadToSend, context.ReceivedPayload, "The large payload should be marshalled correctly using the databus");
+        }
+
+        const int PayloadSize = 100;
+
+        public class Context : ScenarioContext
+        {
+            public byte[] ReceivedPayload { get; set; }
+        }
+
+        public class Sender : EndpointConfigurationBuilder
+        {
+            public Sender()
+            {
+                EndpointSetup<DefaultServer>(builder =>
+                {
+                    builder.Conventions()
+                        .DefiningCommandsAs(t => t.Namespace != null && t.FullName == typeof(MyMessageWithLargePayload).FullName)
+                        .DefiningDataBusPropertiesAs(t => t.Name.Contains("Payload"));
+                    builder.UseDataBus<FileShareDataBus>().BasePath(@".\databus\sender");
+                    builder.UseSerialization<JsonSerializer>();
+                })
+                .AddMapping<MyMessageWithLargePayload>(typeof(Receiver))
+                .ExcludeType<MyMessageWithLargePayload>(); // remove that type from assembly scanning to simulate what would happen with true unobtrusive mode
+            }
+        }
+
+        public class Receiver : EndpointConfigurationBuilder
+        {
+            public Receiver()
+            {
+                EndpointSetup<DefaultServer>(builder =>
+                {
+                    builder.Conventions()
+                        .DefiningCommandsAs(t => t.Namespace != null && t.FullName == typeof(MyMessageWithLargePayload).FullName)
+                        .DefiningDataBusPropertiesAs(t => t.Name.Contains("Payload"));
+
+                    builder.UseDataBus<FileShareDataBus>().BasePath(@".\databus\sender");
+                    builder.UseSerialization<JsonSerializer>();
+                    builder.RegisterComponents(c => c.ConfigureComponent<Mutator>(DependencyLifecycle.InstancePerCall));
+                });
+            }
+
+            public class MyMessageHandler : IHandleMessages<MyMessageWithLargePayload>
+            {
+                public Context Context { get; set; }
+
+                public Task Handle(MyMessageWithLargePayload messageWithLargePayload, IMessageHandlerContext context)
+                {
+                    Context.ReceivedPayload = messageWithLargePayload.Payload;
+
+                    return Task.FromResult(0);
+                }
+            }
+
+            public class Mutator : IMutateIncomingTransportMessages
+            {
+                public Task MutateIncoming(MutateIncomingTransportMessageContext context)
+                {
+                    if (context.Body.Length > PayloadSize)
```

Comment:
It is an additional assert essentially. Happy to remove it. Copy paste from the previous test